app/commands/v2/unpublish.rb

```diff
@@ -132,7 +132,9 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
```

Comment:
I'm fairly sure that you can have more than one, e.g this workflow:

1) create draft
2) publish
3) unpublish
4) redraft
5) publish

In this case, you'll have both an unpublished content item and a published content item (with a higher user-facing version).